### DISCUSSION TITLE
Optionally include Copyrights/Notices in scan results

### DISCUSSION THREAD

**Discussion by anders-swanson** [aquasecurity/trivy#4374] [Ideas]
Is it possible for Trivy to include Copyrights and/or Notice texts in scan results, if they are present? This could be a helpful compliance feature.

**Comment by omkarparkhe** [comment]
I agree with @anders-swanson. It would be very help in terms license compliance and also benefits the user/developer to readily consume a notices file with all the relevant licenses and copyright data from the SBOM generated by Trivy.

**Comment by PeterBurner** [comment]
I also think this would be a valuable feature.

Many licenses contain a copyright notice. It usually consists of a copyright holder and a year or version number. Most of these licenses (like MIT, Apache, GPL, LGPL, MPL, etc.) do require you to reproduce the copyright notice(s) of the original authors and any subsequent contributors when you distribute the software. This is usually a core condition of the license.
Popular SBOM formats even specify a dedicated field for the copyright notice (CycloneDX --> `Copyright`, SPDX --> `copyrightText`, ...).

Unfortunately Trivy currently does not exposed this information in any way.

**Comment by BjoernSaja** [comment]
Basically these parts:
<a href="https://www.youtube.com/live/8hbkfKAl4hk?si=eGDprf78ILv0XS3A&t=819"><img width="1883" height="1057" alt="CleanShot 2025-10-17 at 16 16 34" src="https://github.com/user-attachments/assets/9c820065-79bf-4360-81e4-7df605b800f5" /></a>

**Reply by bilalahmad333** [reply]
I would second @BjoernSaja , this would really helpful feature and adds great value to the final output
